File: Programmers/Lv0/저주의숫자3.py
def solution(n):
    answer = 0
    for i in range(n):
        answer += 1 # 진짜 수를 더하여 만드는 곳
        while answer % 3 == 0 or '3' in str(answer): # 3을 쓸 수 없으니 1씩 더 더해주는 곳
            answer += 1
    return answer

File: Programmers/Lv0/저주의숫자3.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n) {
    int answer = 0;
    for (int i = 0; i < n; i++) {
        ++answer;
        string aString = to_string(answer);
        while (answer % 3 == 0 || aString.find("3") != string::npos)
        {
            ++answer;
            aString = to_string(answer);
        }
    }
    return answer;
}